app/components/course-page/configure-github-integration-modal.hbs

```diff
@@ -6,7 +6,7 @@
   {{#if this.isLoading}}
     <div class="flex items-center mt-3">
       <img alt="Loading icon" src={{this.rippleSpinnerImage}} class="w-4 h-4" />
-      <span class="ml-3 text-gray-700">Loading...</span>
+      <span class="ml-3 text-gray-700 dark:text-gray-200">Loading...</span>
```

Comment:
Doesn't this need to be applied to line 23 and line 36 too? (where we're using `text-gray-600`)

Please compare your implementation to the screenshot in the task description and verify that we've actually fixed the dark mode issue highlighted!